src/main/java/Contacts0.java

```diff
@@ -3,14 +3,12 @@
 
 public class Contacts0 {
 
+    public static final int MAX_RECORD = 100;
+
     public static void main(String[] args) {
         final Scanner SCANNER = new Scanner(System.in);
-        System.out.println("|| ===================================================");
-        System.out.println("|| ===================================================");
-        System.out.println("|| Contacts - Version 0.0");
-        System.out.println("|| Welcome to Contacts!");
-        System.out.println("|| ===================================================");
-        String[][] list = new String[100][3];
+        showWelcomeScreen();
```

Comment:
Good job on extracting the lines to a method hehe xd

Reply:
thank you.